doc/guides/coding_guidelines/index.rst

```diff
@@ -823,3 +823,96 @@ Excluding declarations from the header based on compile-time options may prevent
 their documentation from being generated. Their absence also prevents use of
 ``if (IS_ENABLED(CONFIG_FOO)) {}`` as an alternative to preprocessor
 conditionals when the code path should change based on the selected options.
+
+Rule A.2: Inclusive Language
+============================
+
+Severity
+--------
+
+Required
+
+Description
+-----------
+
+Do not introduce new usage of offensive terms listed below. This rule applies
+but is not limited to source code, comments, documentation, and branch names.
+Replacement terms may vary by area or subsystem, but should aim to follow
+updated industry standards when possible.
+
+Exceptions are allowed for maintaining existing implementations of industry
+standard specifications governed externally to the Zephyr Project.
+
+Existing usage is recommended to change as soon as updated industry standard
+specifications become available, or immediately if no specifications apply.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Offensive Terms
+     - Recommended Replacements
+
+   * - ``master / slave``
+     - - ``{primary, main} / {secondary,replica,subordinate}``
```

Comment:
While we're at it, I reckon the term `subordinate` may not be inclusive (or 100% politically correct) enough.

p.s. spurious space after comma

```suggestion
     - - ``{primary,main} / {secondary,replica,subordinate}``
```